summertime/model/single_doc/multilingual/mbart_model.py

```diff
@@ -112,9 +112,10 @@ def show_capability(cls) -> None:
             "Introduced in 2020, a multilingual variant of BART (a large neural model) "
             "trained on web crawl data.\n"
             "Strengths: \n - Multilinguality: supports 50 different languages\n"
+            " - Higher max input length than mT5 (1024)"
             "Weaknesses: \n - High memory usage"
             "Initialization arguments: \n "
             "- `device = 'cpu'` specifies the device the model is stored on and uses for computation. "
-            "Use `device='gpu'` to run on an Nvidia GPU."
+            "Use `device='cuda'` to run on an Nvidia GPU."
```

Comment:
Oh, no. I think this typo is actually common across all our models... Good catch! 

But do you mind fixing the others as well?

Reply:
will fix!